app/modules/ads/setup.js

```diff
@@ -327,6 +328,12 @@ export const adsSetup = {
       }
     }
 
+    if (context.get('options.nonLazyIncontents.enabled')) {
+      context.set('options.useTopBoxad', true);
```

Comment:
Maybe it's a good decision to do top_boxad on mobile cleanup first or in this ticket?

Reply:
I will create a separate ticket for this